src/controller/adminController.js

```diff
@@ -130,5 +131,44 @@ const applyLeave = async function (req, res) {
     }
 }
 
+const login = async function (req, res) {
+    try{
+        const data = req.body;
+        if(Object.keys(data).length === 0){
+            return res.status(400).send({ message: "please provide data" });
+        }
+        const {Email,Password} = data;
+        if(!isValid(Email)){
+            return res.status(400).send({ message: "please provide Email" });
+        }
+        const findEmail = await admin.findOne({Email:Email});
```

Comment:
u can write 
const findEmail = await admin.findOne({Email});

Reply:
- [x] okk I am checking on to it